spec/static/spec/v0.9/openapitransactional.yaml

```diff
@@ -785,7 +793,12 @@ paths:
       - Asset Upload Management
       summary: Create a new Asset's multipart upload
       description: |
-        Create a new Asset's multipart upload.
+        Create Asset's multipart upload.
+
+        A file part must be at least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
+
+        *Note: in order to provide integrity check during the upload, the base64-encoded 128-bit MD5 digest of each part must
```

Comment:
must + a _be_ computed (picky, I know, sorry :wink:)

Reply:
Not picky but right :-) I'll fix it in next PR though.